### PR TITLE
Keptn API page was missing instructions for getting token

### PR DESCRIPTION
**Pull request by johannes-b**

### CODE REVIEW

File: content/docs/0.6.0/reference/cli/index.md [hunk 1]
```diff
@@ -111,9 +109,9 @@ keptn configure domain YOUR.CUSTOM.DOMAIN
 
 ### keptn auth 
 
-Before the keptn CLI can be used, it needs to be authenticated against a keptn cluster. Therefore, an endpoint and an API token are required.
+Before the Keptn CLI can be used, it needs to be authenticated against a Keptn cluster. Therefore, an endpoint and an API token are required.
```
Comment (christian-kreuzberger-dtx) [inline content/docs/0.6.0/reference/cli/index.md:112]: I thought if we are talking about Keptn as the CLI, we use a lowercase k?

Comment (johannes-b) [reply content/docs/0.6.0/reference/cli/index.md]: That is true. When talking about the keptn CLI, we use lower case k. I fixed it.

File: content/docs/0.6.0/reference/api/index.md [hunk 2]
```diff
@@ -9,8 +9,9 @@ In this section, the functionality and commands of the Keptn REST API are descri
 
 ## Prerequisites
 
-- To access the Keptn API, a running Keptn installation is needed. If you have not set up Keptn yet, [please start here](../../installation/setup-keptn).
+- To access the Keptn API, a running Keptn installation is needed. If you have not set up Keptn yet, please start [here](../../installation/setup-keptn).
 
+- To get the **API token** for the basic authentication of API calls, please see [here](../cli/#keptn-auth).
```
Comment (agrimmer) [inline content/docs/0.6.0/reference/api/index.md:14]: Actually, the API token provides no "basic authentication".

Comment (johannes-b) [reply content/docs/0.6.0/reference/api/index.md]: Valid point. Fixed the wording.

Comment (agrimmer) [reply content/docs/0.6.0/reference/api/index.md]: Thx!

**Pull request merged**